pkgs/desktops/plasma-5/kwallet-pam.nix

```diff
@@ -4,7 +4,8 @@ mkDerivation {
   name = "kwallet-pam";
   nativeBuildInputs = [ extra-cmake-modules ];
   buildInputs = [ pam socat libgcrypt qtbase kwallet ];
-  postPatch = ''
-    sed -i pam_kwallet_init -e "s|socat|${lib.getBin socat}/bin/socat|"
+  qtWrapperArgs = [ "--prefix PATH : ${lib.getBin socat}/bin" ];
```

Comment:
Can we add a `dontWrapQtApps = true;`?
Just to make it clear `pam_kwallet_init` is a script and we have to wrap manually.